media_driver/linux/ult/ult_app/driver_loader.h

```diff
@@ -87,7 +87,7 @@ struct DriverSymbols
     void Clear()
```

Comment:
Folks, I strongly suggest to remove constructor and Clear() entirely, You have class with all public members C++11 already has a way to reset a structure to default value-initialization which is: m_drvSyms = {}.

Reply:
Thank you for catching this, "m_drvSyms = {}" is much better than a Clear function.